app/src/main/res/layout/document__fragment__edit.xml

```diff
@@ -38,27 +37,21 @@
                 android:layout_weight="0"
                 android:text="" />
 
-            <FrameLayout
```

Comment:
This framelayout was causing some issues. Removing it and just using the parent linearlayout